### PR TITLE
remove drop_last from explicit params in FilteredBase.databunch

### PR DESCRIPTION
**Pull request by radekosmulski**
No longer pass `drop_last` as keyword argument to `DataBunch` call in `FilteredBase.databunch`

This addresses part of #220

### CODE REVIEW

**Comment by radekosmulski** [comment]
Had to push another commit - something funny there happened I believe where maybe the notebook for the jekyll exporter was changed but only a single regeneration of scripts was run (I think in such a case one probably needs to run the generation of nbs twice). Have not verified so this is somewhat a wild guess :slightly_smiling_face: 

Anyhow, I think things should now be in order.

**Comment by sgugger** [comment]
`drop_last` should always be set to `True` on the training dataloader, otherwise models with batchnorm layers won't train properly (the last batch will make training fail if it has one element, and give unstable results if it has few elements), so we can't remove it entirely.

**Comment by radekosmulski** [comment]
Thank you Sylvain for the explanation, makes a lot of sense! :slightly_smiling_face: 

That means I have been constructing the DataLoader to predict on incorrectly. I understand now why we use the setting whether a dataset should be getting shuffled or not as a proxy for whether we will be training on it or using it for inference.

Thx a lot again for your explanation!
